pages/common/lp.md

```diff
@@ -0,0 +1,31 @@
+# lp
+
+> print files
+
+- List printers present on the machine.
+
+`lpstat -p | awk '{print $2}`
+
+- List the default printer.
+
+`lpstat -d`
+
+- Print the output of a command.
+
+`echo "test" | lp -d {{printer_name}}`
+
+- Print to the default printer.
```

Comment:
Maybe make the description explicit, i.e. "Print a file to the default printer."